lib/app/routes/core.rb

```diff
@@ -143,6 +147,7 @@ def planned_languages
         def tracks
           ExercismWeb::Presenters::Tracks
```

Comment:
Hey Katrina,
We understand. We think this would mean that after deleting active, upcoming, and planned. We would change the code where active, upcoming, and planned is being called, right? Just checking.